src/tube_manager.c

```diff
@@ -176,6 +174,7 @@ _tube_manager_init(tube_manager* m,
     m->timer_q = gpriority_queue_create(&paged_binary_heap_ctx, _timer_del);
     if (!m->timer_q)
     {
+      LS_ERROR(err, LS_ERR_NO_MEMORY);
```

Comment:
I hope this is the only way that `gpriority_queue_create` can fail, but since the vendor doesn't believe in errors, there's no way to tell.